<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-077G3EM" name="com.RingPayScripts.RingPolicy_L3_Scripts" tests="1" failures="0" timestamp="2023-01-30T20:29:38 IST" time="21.970" errors="1">
  <testcase name="Ringpolicy_L3" time="21.970" classname="com.RingPayScripts.RingPolicy_L3_Scripts">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.business.RingPay_RingPolicy_L3.RegisterUser_Login.login_Positive(RegisterUser_Login.java:46)
at com.RingPayScripts.RingPolicy_L3_Scripts.Ringpolicy_L3(RingPolicy_L3_Scripts.java:128)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- Ringpolicy_L3 -->
  <system-out/>
</testsuite> <!-- com.RingPayScripts.RingPolicy_L3_Scripts -->
